packages/vue-cli-plugin-vuetify/generator/templates/v3/src/plugins/vuetify.js

```diff
@@ -1,6 +1,10 @@
 // Styles
 import '@mdi/font/css/materialdesignicons.css'
+<%_ if (useVite) { _%>
```

Comment:
This doesn't have anything to do with vite, it's the same for both vite and webpack.